src/commands/helpcenter.js

```diff
@@ -0,0 +1,51 @@
+const { SlashCommandBuilder } = require('@discordjs/builders');
+const { MessageEmbed } = require('discord.js');
+
+module.exports = {
+  data: new SlashCommandBuilder()
+    .setName('helpcenter')
+    .setDescription('Provies information to access the helcenter!')
+    .addStringOption((option) =>
```

Comment:
Instead of a `StringOption` here (where users can write anything), I'd prefer if we limited user input to either True or False.
Let's convert this to a `BooleanOption`! 😄 

When done, it should look more like this:

![image](https://user-images.githubusercontent.com/44326005/135725976-715bec47-a11c-4ca9-86b2-f734af7a885f.png)


This isn't something that's documented in the guide or documentation as far as I can find, but the API supports it.
If you need any help researching or executing this, feel free to reach out and I can point you in the right direction or walk through it with you.

Reply:
Thanks for the feedback @aedwardg ! 😄 
I think I've got that working:
![image](https://user-images.githubusercontent.com/78568904/135726176-ae85d4ba-6301-41dd-a5d5-ba7c758fbdc0.png)

Reply:
Awesome! 💪 

Go ahead and push your changes at your convenience and re-request a review from me so that I get the notification when you're ready 🙂